<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debugger::Internal::AttachExternalDialog</class>
 <widget class="QDialog" name="Debugger::Internal::AttachExternalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Debugger</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pidLabel">
       <property name="text">
        <string>Attach to &amp;process ID:</string>
       </property>
       <property name="buddy">
        <cstring>pidLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pidLineEdit"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Utils::FilterLineEdit" name="filterWidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="toolchainLabel">
       <property name="text">
        <string>&amp;Tool chain:</string>
       </property>
       <property name="buddy">
        <cstring>toolchainComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Debugger::Internal::DebuggerToolChainComboBox" name="toolchainComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="procView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::FilterLineEdit</class>
   <extends>Utils::FancyLineEdit</extends>
   <header location="global">utils/filterlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Debugger::Internal::DebuggerToolChainComboBox</class>
   <extends>QComboBox</extends>
   <header>debuggertoolchaincombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
